### PR TITLE
updating Unique Tags screenshot to reflect customer experience

### PR DESCRIPTION
**Pull request by SharpRake**
## Type of change

Quick update to the Unique Tags doc's screenshot to make it align with what customers will actually see.

### What should this PR do?

Resolves https://github.com/chainguard-dev/customer-issues/issues/1185

### Why are we making this change?

I mistakenly used a special private repo in my examples, this updates the examples to show what customers will actually see. 

### How should this PR be tested?

No major content changes, just a quick review here.

### CODE REVIEW

File: content/chainguard/chainguard-images/images-features/unique-tags/index.md [hunk 1]
```diff
@@ -51,9 +51,9 @@ By toggling this button on, each individual Image's details page will show the U
...
+![Screenshot of a portion of the busybox Image's Versions tab. This screenshot shows the ten most recently built versions, some of which include unique tags such as "1-202312280512"](unique-tags-2.png)
 
-Here there are a number of Image versions with tags similar to `:latest-20230626`. You can read this as "latest as of June 26, 2023," meaning that this particular version of the Image was last updated on June 26, 2023. You can use this version's **Pull URL** (`cgr.dev/chainguard-private/busybox:latest-20230626`) to download this Image, and you can be confident that this Pull URL will always refer to the same Image.
+Here there are a number of Image versions with tags similar to `:1-202312280512`. YThis means that this particular version of the Image was last updated on December 28, 2023 at 5:12 AM. You can use this version's **Pull URL** (`cgr.dev/$ORGANIZATION/busybox:1-202312280512`) to download this Image, and you can be confident that this Pull URL will always refer to the same Image.
```
Comment (ltagliaferri) [inline content/chainguard/chainguard-images/images-features/unique-tags/index.md:56]:
```suggestion
Here there are a number of Image versions with tags similar to `:1-202312280512`. This means that this particular version of the Image was last updated on December 28, 2023 at 5:12 AM. You can use this version's **Pull URL** (`cgr.dev/$ORGANIZATION/busybox:1-202312280512`) to download this Image, and you can be confident that this Pull URL will always refer to the same Image.
```

**Review by ltagliaferri — APPROVED** [review]
One typo, thanks for getting to this so quickly!

File: content/chainguard/chainguard-images/images-features/unique-tags/index.md (hunk 1 above)
Comment (SharpRake) [reply content/chainguard/chainguard-images/images-features/unique-tags/index.md:56]: :facepalm:

**Comment by amdawson** [comment]
nice, thank you!

**Pull request merged**